back/spec/acceptance/initiatives_spec.rb

```diff
@@ -427,104 +442,138 @@
       parameter :topic_ids, 'Array of ids of the associated topics'
       parameter :area_ids, 'Array of ids of the associated areas'
       parameter :assignee_id, 'The user id of the admin that takes ownership. Only allowed for admins.'
+      parameter :anonymous, 'Post this initiative anonymously - true/false'
     end
     ValidationErrorHelper.new.error_fields(self, Initiative)
 
-    let(:id) { @initiative.id }
-    let(:location_point_geojson) { { type: 'Point', coordinates: [51.4365635, 3.825930459] } }
-    let(:location_description) { 'Watkins Road 8' }
-    let(:header_bg) { file_as_base64 'header.jpg', 'image/jpeg' }
-    let(:topic_ids) { create_list(:topic, 2).map(&:id) }
-    let(:area_ids) { create_list(:area, 2).map(&:id) }
+    describe 'published initiatives' do
+      let(:id) { @initiative.id }
+      let(:location_point_geojson) { { type: 'Point', coordinates: [51.4365635, 3.825930459] } }
+      let(:location_description) { 'Watkins Road 8' }
+      let(:header_bg) { file_as_base64 'header.jpg', 'image/jpeg' }
+      let(:topic_ids) { create_list(:topic, 2).map(&:id) }
+      let(:area_ids) { create_list(:area, 2).map(&:id) }
+
+      describe do
+        let(:title_multiloc) { { 'en' => 'Changed title' } }
+
+        example_request 'Update an initiative' do
+          expect(status).to be 200
+          json_response = json_parse(response_body)
+          expect(json_response.dig(:data, :attributes, :title_multiloc, :en)).to eq 'Changed title'
+          expect(json_response.dig(:data, :attributes, :location_point_geojson)).to eq location_point_geojson
+          expect(json_response.dig(:data, :attributes, :location_description)).to eq location_description
+          expect(json_response.dig(:data, :relationships, :topics, :data).pluck(:id)).to match_array topic_ids
+          expect(json_response.dig(:data, :relationships, :areas, :data).pluck(:id)).to match_array area_ids
+        end
+
+        example 'Check for the automatic creation of an upvote by the author when the publication status of an initiative is updated from draft to published', document: false do
+          @initiative.update(publication_status: 'draft')
```

Comment:
I would have a preference for `update!` over `update`, to guarantee that the update won't silently fail.